paraview/xmls/DistanceMatrixDistortion.xml

```diff
@@ -161,6 +172,7 @@
       ${DEBUG_WIDGETS}
 
       <PropertyGroup panel_widget="Line" label="Input options">
+        <Property name="DoNotNormalize" />
```

Comment:
This is more of an output option I would say.
You may want to create a new, similar `PropertyGoup` with label "Output options" and put this option in it.